tests/test_data/boltz_results_boltz_test/lightning_logs/version_0/hparams.yaml

```diff
@@ -0,0 +1,116 @@
+atom_s: 128
```

Comment:
What this?

Reply:
Do we need all this data?

Reply:
This one no, This is the raw output of the boltz run. I don't think we need to paramaters so I'll remove that and I'll remove more later when I know if they are definitely not needed.